app/services/feature.rb

```diff
@@ -14,6 +14,7 @@ def initialize_feature_flags
         Flipper.add(feature_flag_key)
       end
       Flipper.enable(:maths_npq)
+      Flipper.disable(:targeted_support_funding)
```

Comment:
I think it will disable the feature flag every time the code reloads.

Reply:
Yes, that is intentional as far as i am concerned. Might be worth it to test this and check with jake.